nautobot/extras/migrations/0072_migrate_custom_field_data.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 3.2.18 on 2023-03-17 17:25
+
+from django.db import migrations
+from nautobot.core.models.fields import slugify_dashes_to_underscores
+
+
+def generate_unique_custom_field_slug_and_migrate_custom_field_data(apps, schema_editor):
+    CustomField = apps.get_model("extras", "customfield")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    # Make sure that the key is unique during migration
+    cf_keys = []
+    for custom_field in CustomField.objects.all().order_by("created"):
+        original_cf_key = custom_field.key
+        cf_key = original_cf_key
+        append_counter = 2
+        while cf_key in cf_keys:
+            cf_key_append = f"_{append_counter}"
+            max_key_length = 50 - len(cf_key_append)
+            cf_key = original_cf_key[:max_key_length] + cf_key_append
+            append_counter += 1
+        if cf_key != original_cf_key:
+            print(
+                f'  CustomField Class "{CustomField.key}" key "{original_cf_key}" is not unique, changing to "{cf_key}".'
```

Comment:
Is `{CustomField.key}` the correct thing to be printing here?

Reply:
Ur right should be `label`.